charts/cryptpad/templates/volume.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: data
+spec:
+  capacity:
+    storage: {{ .Values.volumeSize }}
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Recycle
+  storageClassName: ""
+  hostPath:
+    path: /persistent-data/
```

Comment:
I think if we deploy more than one instance, it will have the same content? or it's supposed to deploy only one per cluster?